nautobot/circuits/models.py

```diff
@@ -35,7 +35,9 @@ class ProviderNetwork(PrimaryModel):
     name = models.CharField(max_length=100, db_index=True)
     slug = AutoSlugField(populate_from="name")
     provider = models.ForeignKey(to="circuits.Provider", on_delete=models.PROTECT, related_name="provider_networks")
+    # todoindex:
```

Comment:
circuits/filters.py-95-        if not value.strip():
circuits/filters.py-96-            return queryset
circuits/filters.py:97:        return queryset.filter(
circuits/filters.py-98-            Q(name__icontains=value) | Q(description__icontains=value) | Q(comments__icontains=value)
circuits/filters.py-99-        ).distinct()